7-layouts/style.css

```diff
@@ -0,0 +1,89 @@
+body {
+    color: whitesmoke;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+footer {
```

Comment:
always prefer classnames over tag names so you avoid side effects and forcing your markup to match your styles.